### PR TITLE
mrcll - 13 done

### PR DESCRIPTION
**Pull request by remrcll**
Pls. check.

### CODE REVIEW

**Comment by Ungoot** [comment]
Good job

File: 13-PB-language-boolean/13-mrcll-solution.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+// What would happen if we were to use =?
+// Answer: 
+//It would give us an error, because = stands for assignment.
+
+
+// 2. Not
+// Store the value true to a variable. Using a ternary operator,print the false option by checking the value of the initial variable.
+
+let truthy = true
+console.log("Answer 2: " +  (truthy != true ? true : false));
```
Comment (Ungoot) [inline 13-PB-language-boolean/13-mrcll-solution.js:31]: please make it a habit to use === and !== for comparison

Comment (remrcll) [reply 13-PB-language-boolean/13-mrcll-solution.js]: fixed

Comment (Ungoot) [reply 13-PB-language-boolean/13-mrcll-solution.js]: Thank you

**Pull request closed without merging**